ridgeback_baxter_description/launch/ridgeback_baxter_world.launch

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<launch>
+
+  <!-- Load the combined ridgeback-baxter URDF's to the param server -->
+  <param name="robot_description"
+         command="$(find xacro)/xacro.py --inorder
+         $(find ridgeback_baxter_description)/urdf/ridgeback_baxter.urdf.xacro gazebo:=true"/>
+
+  <!-- Launch baxter_simulator -->
+  <include file="$(find baxter_gazebo)/launch/baxter_world.launch">
+    <arg name="load_robot_description" value="false"/>
+  </include>
+
+  <!-- Launch ridgeback_simulator -->
+  <!-- TODO -->
+
```

Comment:
I am fine with this simulator launch file being placed elsewhere that makes more sense. I just put it here as a placeholder.